modules/ROOT/pages/bolt/message.adoc

```diff
@@ -395,6 +396,11 @@ Clients wishing to retry initialization should establish a new connection.
 
 The `routing::Dictionary(address::String)` field is added in version *4.1* and adds an indicator if the server should carry out routing, according to the given routing context.
 
+The `notifications_minimum_severity::String notifications_disabled_categories::List<String>` field is added in version *5.2* and adds the ability to configure notifications for the given connection
+
+The `notifications_minimum_severity` specifies the minimum severity a notification needs to have to be returned. Please see the Cypher documentation for acceptable entries.
```

Comment:
do we have a link to the relevant page in the cypher docs?

Reply:
This maybe? https://neo4j.com/docs/status-codes/current/notifications/#notification-categories

The thing is that this feature is scheduled for server version 5.7, so there's no updated documentation yet.